packages/graph/hash_graph/lib/graph/src/store/query/filter.rs

```diff
@@ -129,161 +128,97 @@ impl<'p> Filter<'p, Entity> {
         ])
     }
 
-    /// Creates a `Filter` to search for a specific entity edition, identified by its
-    /// [`EntityVertexId`].
+    /// Creates a `Filter` to search for linked entities where the specified [`EntityId`] identifies
+    /// the source [`Entity`].
     #[must_use]
-    pub fn for_entity_by_vertex_id(vertex_id: EntityVertexId) -> Self {
-        // TODO: Adjust structural queries for temporal versioning
-        //   see https://app.asana.com/0/0/1203491211535116/f
-        Self::All(vec![
-            Self::Equal(
-                Some(FilterExpression::Path(EntityQueryPath::OwnedById)),
-                Some(FilterExpression::Parameter(Parameter::Uuid(
-                    vertex_id.base_id().owned_by_id().as_uuid(),
-                ))),
-            ),
-            Self::Equal(
-                Some(FilterExpression::Path(EntityQueryPath::Uuid)),
-                Some(FilterExpression::Parameter(Parameter::Uuid(
-                    vertex_id.base_id().entity_uuid().as_uuid(),
-                ))),
-            ),
-            Self::Equal(
-                Some(FilterExpression::Path(EntityQueryPath::ProjectedTime)),
-                Some(FilterExpression::Parameter(Parameter::Timestamp(
-                    vertex_id.version(),
-                ))),
-            ),
-        ])
-    }
-
-    /// TODO
-    #[must_use]
-    pub fn for_outgoing_link_by_source_entity_vertex_id(vertex_id: EntityVertexId) -> Self {
-        // TODO: Adjust structural queries for temporal versioning
-        //   see https://app.asana.com/0/0/1203491211535116/f
+    pub fn for_outgoing_link_by_source_entity_id(entity_id: EntityId) -> Self {
         Self::All(vec![
             Self::Equal(
                 Some(FilterExpression::Path(EntityQueryPath::LeftEntity(
                     Box::new(EntityQueryPath::OwnedById),
                 ))),
                 Some(FilterExpression::Parameter(Parameter::Uuid(
-                    vertex_id.base_id().owned_by_id().as_uuid(),
+                    entity_id.owned_by_id().as_uuid(),
                 ))),
             ),
             Self::Equal(
                 Some(FilterExpression::Path(EntityQueryPath::LeftEntity(
                     Box::new(EntityQueryPath::Uuid),
                 ))),
                 Some(FilterExpression::Parameter(Parameter::Uuid(
-                    vertex_id.base_id().entity_uuid().as_uuid(),
-                ))),
-            ),
-            Self::Equal(
-                Some(FilterExpression::Path(EntityQueryPath::LeftEntity(
-                    Box::new(EntityQueryPath::ProjectedTime),
-                ))),
-                Some(FilterExpression::Parameter(Parameter::Timestamp(
-                    vertex_id.version(),
+                    entity_id.entity_uuid().as_uuid(),
                 ))),
             ),
         ])
     }
 
-    /// TODO
+    /// Creates a `Filter` to search for linked entities where the specified [`EntityId`] identifies
```

Comment:
Like above, probably more clear to make it `incoming linked entities`

Reply:
75bc0acc3b6894c1bf282105e1b19bade510c748